frontend/components/FooterCom.vue

```diff
@@ -1,20 +1,13 @@
 <template>
-  <div class="footer">
-    <v-app-bar color="#0079CF" />
-  </div>
+  <v-footer
+    color="#0079CF"
```

Comment:
Gebruik een config file zodat er een centrale plek is waar je al je kleuren kan beheren